### DISCUSSION TITLE
Possibility to use with erofs instead of squashfs?

### DISCUSSION THREAD

**Discussion by andreadaoud** [rauc/rauc#960] [General]
Hello,

In verity mode, is it possible to use erofs rather than squashfs? The official document only mentions squashfs now.

**Comment by ejoerns** [comment]
Verity is the bundle format and is not related to your device's file system(s) in any way.

For the bundle format, SquashFS is fixed and there is no need to change it I guess.

You are free to install any file system with RAUC and also to use or not use dm-verity as the underlying device mapper. Using dm-verity and erofs is transparent to RAUC. Just make sure to use "*.img" image suffix (image type) and "raw" as slot type in the `system.conf`.
